src/status_im2/contexts/syncing/scan_sync_code/view.cljs

```diff
@@ -119,12 +121,7 @@
         :data           [{:id 1 :label (i18n/label :t/scan-sync-qr-code)}
                          {:id 2 :label (i18n/label :t/enter-sync-code)}]
         :on-change      (fn [id]
-                          (reset! active-tab id)
-                          (reset! read-qr-once? false))}]]]))
-
-(defn- header
-  [props]
-  [:f> f-header props])
```

Comment:
We don't need this wrapper, we use `:f>` wrappers to expose an API, for internal component use, we can directly use  the `:f>` call